CatAnalyzer/test/customise_cff.py

```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
have u tested this with create-batch?
i dont think the correct process.source.fileNames will be read